### PR TITLE
DOC-3983: add HFE examples to the hash tutorial

### PR DESCRIPTION
**Pull request by dwdougherty**

### CODE REVIEW

**Comment by dwdougherty** [comment]
This PR has been open for 3 weeks. Please review so I can close my ticket and OKR.

File: src/ds/hashes.md [hunk 1]
```diff
@@ -44,4 +44,39 @@ HINCRBY bike:1 price 100
 HINCRBY bike:1 price -100
 ```
 
+New in Redis Community Edition version 7.4 is the ability set the expiration time or the remaining time-to-live (TTL) for individual hash fields. This is called hash field expiration (HFE). HFE works just like [key expiration](https://redis.io/docs/latest/develop/use/keyspace/#key-expiration) and includes the following commands:
```
Comment (ViktarStarastsenka) [inline src/ds/hashes.md:47]: New in [Redis Community Edition version 7.4](https://hub.docker.com/layers/redis/redis-stack/7.4.0-v0/images/sha256-3e3c86603a81712d1311bc619ad124de15b2dca2b50722f23a4502b4d4054ba8) is the ability set the expiration time or the remaining time-to-live (TTL) for individual hash fields. This is called hash field expiration (HFE). HFE works just like [key expiration](https://redis.io/docs/latest/develop/use/keyspace/?utm_source=redisinsight&utm_medium=main&utm_campaign=tutorials#key-expiration) and includes the following commands:

**Review by ViktarStarastsenka — CHANGES_REQUESTED** [review]

File: src/ds/hashes.md (hunk 1 above)
Comment (dwdougherty) [reply src/ds/hashes.md]: Thank you, @ViktarStarastsenka .  Good suggestions. Merged.

**Review by ViktarStarastsenka — APPROVED** [review]
LGTM

**Pull request merged**